Add 2022-04-28 agenda and minutes (#56)
solid/notifications-panel: meetings/2022-04-28.md

* SC: Minor follow-up. Not urgent. Time to integrate this in while reviews from the Editors team is taking place. Can we wrap this up this week so that we can include it part of Notifications Protocol that's currently under review ( https://github.com/solid/specification/pull/391 )?

> * SC: Made a brief comment to move this forward; otherwise we can merge the suggestions. The text between README and spec does not need to be different.

Reviewer: ```suggestion
* SC: Made a brief comment to move this forward; otherwise we can merge the suggestions. The text does not need to be different between README and spec.
```

>* SC: In the W3C process, editors are able/expected to make editorial changes where appropriate

* SC: Can we resolve these actions before 2022-05-04?
* SC: I can take on some of these ACTIONs if not completed by 2022-05-03.

### Define WebPushSubscription2022 type for Solid Notifications
URL: https://github.com/solid/notifications/pull/64

* SC: Please review.
* SC: Let's have proposer in the meeting when we accept/merge provided no objections...
* eP: any outstanding issue can be tracked separately, this is an initial ED.

> * SC: Sure, as long as it is no major change e.g. term/property use.. 

Reviewer: ```suggestion
* SC: Sure, as long as it is no major change, e.g., term/property use...
```

### Protocol doesn't mention Subscription Response
URL: https://github.com/solid/notifications/issues/57

> * eP: Subscription response has RPC model. We provide type, topic etc and server processes and responds. Different subscription types have different behaviour. Sometime properties go on request and some on response. We want clarity on which properties we would like on request and on response.

Reviewer: ```suggestion
* eP: Subscription response has RPC model. We provide type, topic, etc., and server processes and responds. Different subscription types have different behaviour. Properties sometimes go on request and sometimes on response. We want clarity on which properties we would like on request and on response.
```

* SC: This is in response to the POST request right? The response payload.

> * eP: 

Reviewer: delete or fill in
```suggestion
```

> * SC: There are two parts. One is about information relevant to all subscription types e.g., date, subscription type, .. and then there is subscription specific stuff. The former we can look into. The latter is covered by each subscription type.

Reviewer: ```suggestion
* SC: There are two parts. One is about information relevant to all subscription types (date, subscription type, etc.) and the other is subscription-specific stuff. The former we can look into. The latter is covered by each subscription type.
```

* eP: Every sub type uses different property on request and response. 

> * SC: Two concerns: What payload should include? What should be specific to each sub type?

Reviewer: ```suggestion
* SC: Two concerns: What payload should we include? What should be specific to each sub type?
```

* eP: what payload should include is secondary. Clarity wanted on request and response
* SC: Whatever is relevant to the request.

> * RG: So there are some common properties on requests and responses, which should be defined in the core, some other ones are sub type specific.

Reviewer: ```suggestion
* RG: So there are some common properties on requests and responses, which should be defined in the core; some other ones are sub-type specific.
```
